docroot/themes/humsci/humsci_airy/humsci_airy.info.yml

```diff
@@ -0,0 +1,18 @@
+name: HumSci Airy
+type: theme
+base theme: humsci_basic
+description: A light and airy theme.
+package: Humsci
+core: 8.x
+core_version_requirement: ^8 || ^9
```

Comment:
perhaps we just want to limit this to all versions of drupal 8 https://www.drupal.org/node/3070687

Reply:
Yeah, that was what `drush generate` created, but it's probably safer to go with just 8, so I can change that.